### ISSUE TITLE
[Feature Request]: Update the Data in the Projects page.

### ISSUE THREAD

**Issue by ramG-reddy** [bsoc-bitbyte/BSoC-Website#64]
### Is your feature request related to a problem? Please describe.

The current projects page has previous year's projects, some of them are not considered for this year's program, there are also new projects added this year.

### Describe the solution you'd like

Update the data of the projects page with the data of all projects in bsoc'24.

**Comment by rohansen856** [comment]
I would like to work on this issue. I will just get the details of this years project directly from the discord server and update it in the required page.

**Comment by ramG-reddy** [comment]
@rohansen856 Now you can work on it.

**Comment by ramG-reddy** [comment]
Busify-andriod is not a part of this year's projects and the project names of `Leave Application`, `Gate Entry System`, `2D platformer` are different. Rest are fine.

**Comment by rohansen856** [comment]
i have opened pull request #67 for this issue.

**Comment by ramG-reddy** [comment]
I see that you have not changed the maintainers names for the projects, do change them with the new maintainers names. if there are two maintainers, comma separate them.

**Comment by rohansen856** [comment]
i will get the names of the maintainer and update it. thank you.

**Comment by rohansen856** [comment]
[
				{
					id: 1,
					projectName: 'BSoC Website',
					name: 'Parth Singh, Ramgopal Reddy',
					link: 'https://github.com/bsoc-bitbyte/BSoC-Website',
					techStack: 'Vue, Firebase, Bootstrap',
					description:
						'A website for BSoC to showcase all the projects and events that are being conducted under BSoC.',
				},
				{
					id: 2,
					projectName: 'GetIt',
					name: 'Akash Paloju, Kushagra Gupta',
					link: 'https://github.com/bsoc-bitbyte/GetIt',
					techStack: 'Vue, Firebase, Redis, MongoDB, Python',
					description:
						'A single Platform where merchandise of every club of college can be found and will be made available for everyone to buy.',
				},
				{
					id: 3,
					projectName: 'Busify Web',
					name: 'Pratik Solanki, Rishikesh Kupalla',
					link: 'https://github.com/bsoc-bitbyte/busify',
					techStack: 'React, Typescript, MUI, SCSS, NESTJS, Postgres, REDIS',
					description:
						'A centralised bus ticket booking system that will allow the students to book tickets in advance through an online website, saving time and effort.',
				},
				{
					id: 4,
					projectName: 'Alumni Website',
					name: 'Eshaan Modi, Samaksh Agarwal ',
					link: 'https://github.com/BitByte-TPC/alumni',
					techStack: 'HTML, CSS, Javascript, Python',
					description:
						'This web application aims to facilitate the Alumni Affairs of IIITDM Jabalpur',
				},
				{
					id: 5,
					projectName: 'myLeave',
					name: 'Agrim, Aditya',
					link: 'https://github.com/bsoc-bitbyte/myLeave',
					techStack: 'Kotlin, Architecture',
					description:
						'myLeave is a user-friendly application designed to streamline and simplify the process of submitting and managing leave requests within an organization',
				},
				{
					id: 6,
					projectName: 'DigiGate',
					name: 'Agrim, Aditya',
					link: 'https://github.com/bsoc-bitbyte/DigiGate',
					techStack: 'Kotlin, Architecture',
					description:
						'This project aims to enhance gate entry and exit management for better security and entry management, by implementing a QR-based system',
				},
				{
					id: 7,
					projectName: 'Summer Analytics',
					name: 'Joyee Biswas, Shreyans Katariya',
					link: 'https://github.com/bsoc-bitbyte/Summer_of_ML',
					techStack: 'Tensorflow, Seaborn, Natplotlib, Pandas, Numpy ',
					description:
						'Machine Learning made simple. Going right from the traditional Linear Regression to building Neural Networks, including all the nitty gritty optimizations, regularizations and loss functions.',
				},
				{
					id: 8,
					projectName: 'Dreamyard',
					name: 'Akshay Bhel, Prince Bhujetia',
					link: 'https://github.com/bsoc-bitbyte/Dreamyard',
					techStack: 'Unity Engine, C#',
					description:
						'An exhilarating 2D platform game that combines intense platforming challenges with thrilling combat encounters. where you have to fight and tackle enemies to complete the platform.',
				},
			],

**Comment by rohansen856** [comment]
i have fixed the issues mentioned above in pr #69 . thank you
